### PR TITLE
updated stale learn-phragmen

### LINKED ISSUE #1878: [STALE] docs/learn-phragmen.md
Last edit happened 45 days ago 😬

### PR DESCRIPTION
**Pull request by jakehemmerle**
Updated with references to BalPhragmms and fixed some links

should close #1878 and #1511

### CODE REVIEW

File: docs/learn-phragmen.md [hunk 1]
```diff
@@ -610,24 +611,26 @@ an optimal manner.
 
 There are several further restrictions put in place to limit the complexity of the election and
 payout. As already mentioned, any given nominator can only select up to 16 validators to nominate.
-Conversely, a single validator can have only {{ polkadot_max_nominators }} nominators. A drawback to
-this is that it is possible, if the number of nominators is very high or the number of validators is
-very low, that all available validators may be "oversubscribed" and unable to accept more
-nominations. In this case, one may need a larger amount of stake to participate in staking, since
-nominations are priority-ranked in terms of amount of stake.
+Conversely, a single validator can have only 128 nominators. A drawback to this is that it is
```
Comment (Swader) [inline docs/learn-phragmen.md]: Why the change from a dynamic value?

Comment (jakehemmerle) [reply docs/learn-phragmen.md]: It was not rendering properly on the webpage, which I should have fixed instead of just hardcoding it. Any clue how to set it?
Screen Shot 2021-04-26 at 9 57 16 AM

**Comment by Swader** [comment]
See README of the wiki, under conditional rendering. TLDR you define the values in the inject-dict file and the values are injected after mirroring. This might indicate something broke in the build process, or that the keys in the dictionary broke. I will take a look locally too.

File: docs/learn-phragmen.md (hunk 1 above)
Comment (jakehemmerle) [reply docs/learn-phragmen.md]: interesting enough, I see this rendered properly on wiki.polkadot.network, but not localhost. Do I need to know how to reproduce locally? Where can I get a list of possible variables?

**Comment by jakehemmerle** [comment]
Thanks for pointing that out, I see the key/value is there now

**Comment by jakehemmerle** [comment]
unremoved dynamic value for {{ polkadot_max_nominators }}. should be good to go
